README.md

```diff
@@ -448,10 +448,10 @@ ImageSourceMatcher networkSourceMatcher({
   String extension: "your extension",
 }) =>
     (attributes, element) {
-      final src = Uri.parse(attributes["src"]);
+      final src = Uri.parse(attributes["src"] ?? "about:blank");
```

Comment:
Maybe move the `?? "about:blank"` to the `_src` helper method below.

Reply:
That's just in the readme, I don't see a _src helper method in there. 

I edited the readme to "migrate to nullsafety" per se.

Reply:
True - I wasn't looking right.